src/modules/settings/settings.controller.ts

```diff
@@ -0,0 +1,39 @@
+import { ApiPath, PageTitle, PagesPath } from "~/libs/enums/enums.js";
+import { asyncHandler } from "~/libs/helpers/helpers.js";
+import { type HTTPMethod } from "~/libs/modules/http/http.js";
+import { type Application, type Controller } from "~/libs/types/types.js";
+
+class SettingsController implements Controller {
+	private initPages(app: Application) {
+		app.get(
+			PagesPath.SETTINGS,
+			asyncHandler(async (req, res) => {
```

Comment:
I think it makes sense to be able to visit the settings page only when you're logged in. Ensure that this page is not accessible for not authorized users